### ISSUE TITLE
Inputs in dropdown are not accessible via keyboard nav

### ISSUE THREAD

**Issue by mwargan** [picocss/pico#490]
Please search for duplicate or closed issues first.
  
## Describe the issue
Cannot tab into and select radio or checkboxes with keyboard when they are in a dropdown.  

### Current Behavior
Tabs to the next dropdown, skips content inside altogether.

### Expected Behavior
To correctly tab

### Reproduction URL
https://picocss.com/docs/dropdown

### Suggested fix
I've added a `tabindex` to my inputs inside the dropdown. It feels wrong though, just because inputs natively have a `tabindex=0` set.
